index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/71e053ea26.js" crossorigin="anonymous"></script>
+    <title>Document</title>
+</head>
+<body>
+        <header id="header">
+            <nav id="nav-bar">
+                <a href="#header"><img src="Welcome.png" id="Welcome"></a>
+                <a><i class=" fa1 fa-solid fa-bars"></i></a>
+             </nav>
+         </header>
+         <section class="headline">
+            <div id="content">
+                <h1 class="main_content">
+                   Hey there. I'm Mariane <br>  <span id="green-word">I'm a software developer</span>
+                 </h1>
+                 <p class="secondary-text"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don't hestiate to contact me.</p>
+            </div>
+         
+        
+            <div id="div-links">
+                <ul>
+                    <li><a><i class="fa fa-brands fa-twitter twitter"></i></a></li>
+                    <li><a><i class="fa fa-brands fa-linkedin linkedin"></i></a></li>
+                     <li><a><i class="fa fa-solid fa-m m" ></i></a></li>
+                    <li><a><i class="fa fa-brands fa-github github"></i></a></li>
+                    <li><a><i class="fa fa-light fa-hand-peace hand"></i></a></li>
```

Comment:
[OPTIONAL]You did a great job but let's stick to the design and fix the images of the social links.
<img width="205" alt="Screenshot 2023-01-12 at 2 11 22 AM" src="https://user-images.githubusercontent.com/25887253/212153811-554406ef-d569-4542-8584-b483d8045c9d.png">
<img width="205" alt="Screenshot 2023-01-12 at 2 11 29 AM" src="https://user-images.githubusercontent.com/25887253/212153823-abc5c097-2cf3-48f1-9b14-f437cdea283a.png">